README.md

```diff
@@ -45,19 +45,85 @@ These color themes are bundled with all releases of Lite XL by default.
 
 ## Quick Build Guide
 
-If you compile Lite XL yourself, it is recommended to use the script
-`build-packages.sh`:
+To compile Lite XL yourself, you must have the following dependencies installed
+via your desired package manager, or manually.
+
+### Prerequisites
+
+- Meson (>=0.63)
+- Ninja
+- SDL2
+- PCRE2
+- FreeType2
+- Lua 5.4
+- A working C compiler (GCC / Clang / MSVC)
+
+SDL2, PCRE2, FreeType2 and Lua will be downloaded by Meson
+if `--wrap-mode=forcefallback` or `--wrap-mode=default` is specified.
+
+> [!NOTE]
+> MSVC is used in the CI, but MSVC-compiled binaries are not distributed officially
+> or tested extensively for bugs.
+
+On Linux, you may install the following dependencies for the SDL2 X11 and/or Wayland backend to work properly:
+
+- `libX11-devel`
+- `libXi-devel`
+- `libXcursor-devel`
+- `libxkbcommon-devel`
+- `libXrandr-devel`
+- `wayland-devel`
+- `wayland-protocols-devel`
+- `dbus-devel`
+- `ibus-devel`
+
+The following command can be used to install the dependencies in Ubuntu:
 
 ```sh
-bash build-packages.sh -h
+apt-get install python3.8 python3-pip build-essential git cmake wayland-protocols libsdl2-dev
+pip3 install meson ninja
```

Comment:
Avoid going to a new line for copypasting's sake.
```suggestion
apt-get install python3.8 python3-pip build-essential git cmake wayland-protocols libsdl2-dev pip3 install meson ninja
```

Reply:
I'm confused here. The apt-get and pip are suppsed to be different commands.

Reply:
Yeah, my bad, looks like I still need to fully wake up...